bigquery-connector-common/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -514,6 +514,10 @@ public void loadDataIntoTable(
             finishedJob.getJobId());
       }
     } catch (Exception e) {
+      if (finishedJob == null) {
```

Comment:
Can we move this below the if loop just before `JobId jobId = finishedJob.getJobId();`?

Reply:
I initially put it there. Then, checking the logic, the catch clause is implemented when the job is created successfully but failed to finish. Hence, if the `finishedJob` is `null`, then there is no need to check the exception message against the partition type. That's why I moved it to the type so other developers understand its priority.

Reply:
ah. understood. good point